plugin/trino-hive/src/main/java/io/trino/plugin/hive/HiveMetadata.java

```diff
@@ -3804,4 +3804,12 @@ public boolean supportsReportingWrittenBytes(ConnectorSession session, SchemaTab
     {
         return true;
     }
+
+    @Override
+    public boolean supportsMultipleWritersPerPartition(ConnectorSession session, SchemaTableName tableName)
+    {
+        Table table = metastore.getTable(tableName.getSchemaName(), tableName.getTableName())
+                .orElseThrow(() -> new TableNotFoundException(tableName));
+        return !isTransactionalTable(table.getParameters());
```

Comment:
Add explanation.
See https://github.com/trinodb/trino/blob/9928afe12a4931ee51aa60b7e3eeaf52ae2d3957/plugin/trino-hive/src/main/java/io/trino/plugin/hive/HivePageSink.java#L389-L390 for example

Reply:
BTW is it OK to do for non-transactional bucketed tables?
The above code prohibits multiple (sequential) files in such case, so I'd assume multiple concurrent files isn't OK either.